build/cleanup-pr-environments.yml

```diff
@@ -0,0 +1,29 @@
+# DESCRIPTION: 	
+# Removes resource groups used in PRs.
+# THIS WILL DELETE ALL PR RESOURCE GROUPS, EVEN THOSE CURRENTLY RUNNING.
+# Check that no PRs are currently running and no one needs any of the resource groups before running.
```

Comment:
Is there a way to filter by time to avoid deleting a RG of an active run so that we minimize the manual checks?

Reply:
I didn't think so, but after some more looking I think I found a way!